pymatgen/transformations/standard_transformations.py

```diff
@@ -696,13 +696,18 @@ def __init__(self, max_denominator=5, tol=0.25):
         self.max_denominator = max_denominator
         self.tol = tol
 
-    def apply_transformation(self, structure):
+    def apply_transformation(self, structure,fix_denominator=False):
```

Comment:
This breaks the API for apply_transformation, and violates PEP8. If anything, this parameter should be part of the init.

Reply:
Thanks for the comment, fixed that.